scripts/checkMissingTranslations.ts

```diff
@@ -0,0 +1,59 @@
+import { Resource, ResourceKey } from 'i18next'
+import i18n from '../src/i18n'
+
+const checkRecursiveTranslation = (
+  searchingPath: string,
+  defaultLanguageObject: ResourceKey,
+  translatedLanguageObject: ResourceKey,
+) => {
+  if (typeof defaultLanguageObject === 'string' || typeof translatedLanguageObject === 'string') {
+    if (typeof defaultLanguageObject === 'object') {
+      console.log(
+        `📙 Found a string for path ${searchingPath} while is and object for the default language`,
+      )
+    }
+    return
+  }
+  const defaultKeys: string[] = Object.keys(defaultLanguageObject)
+  const translatedKeys: string[] = Object.keys(translatedLanguageObject)
+  if (defaultKeys.length === 0 || translatedKeys.length === 0) {
+    return
+  }
+  defaultKeys.forEach((key) => {
+    if (!translatedLanguageObject[key]) {
+      console.log(`📙 The key ${key} is not present for path ${searchingPath}`)
```

Comment:
I think that we should make this a `console.warn()`. Also, it might be helpful to add some color to these log messages using something like https://www.npmjs.com/package/chalk.

Reply:
Yes I though too, but I didn’t want to add an external library only for this sonI used emoji

Reply:
It will only be a devDepedency, so I don't think adding an external package is a bad thing. We'll just want to make sure that it doesn't have a restrictive license and is relatively stable.

Reply:
Yes of course. I’ll fix this